examples/crm/src/companies/CompanyShow.tsx

```diff
@@ -275,7 +276,12 @@ const DealsIterator = () => {
                             secondary={
                                 <>
                                     {/* @ts-ignore */}
-                                    {stageNames[deal.stage]},{' '}
+                                    {
+                                        dealStages?.find(
```

Comment:
I think you've repeated this at least once, so it'd be a good idea to extract it to a function in the `deal` domain